File: output/csharp/Modules/Scripting/PutScript/PutScriptRequest.cs
using Nest.CommonOptions;
using Nest.Modules;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Modules {

	public class PutScriptRequest : RequestBase {
		
		[DataMember(Name="master_timeout")]
		public Time MasterTimeout { get; set; }


		[DataMember(Name="timeout")]
		public Time Timeout { get; set; }


		[DataMember(Name="script")]
		public StoredScript Script { get; set; }

	}
}

File: specification/specs/modules/scripting/put_script/PutScriptRequest.ts
@rest_spec_name("put_script")
class PutScriptRequest extends RequestBase {
  path_parts?: {
    id: Id;
    context?: Name;
  }
  query_parameters?: {
    master_timeout?: Time;
    timeout?: Time;
  }
  body?: {
    script?: StoredScript;
  }
}
